### ISSUE TITLE
Prośba o aktywację konta sandbox

### ISSUE THREAD

**Issue by JuliuszElsner** [allegro/allegro-api#9395]
Dzień dobry,
proszę o aktywacje konta w allegro sandbox, nazwa konta juliuszelsner.
Link aktywacyjny, który otrzymuje na adres mailowy nie działa, tzn. działa ale prowadzi do produkcyjnej wersji allegro.

**Comment by MartaNowaczyk** [comment]
Dzień dobry, aktywowałam twoje konto.

**Comment by aszymkowiak** [comment]
Dzień dobry, prosiłbym o aktywację konta firmowego. login: agdtech

**Comment by Lukasz-Zurek** [comment]
@aszymkowiak Aktywowałem konto.

**Comment by Zavanven** [comment]
Dzień dobry, również poproszę o aktywację konta sandbox, login: magall
Próbuję przejść przez weryfikację konta bankowego ale pomimo oczekwiania dalej otrzymuję ten sam komunikat
![image](https://github.com/user-attachments/assets/b4c7f979-a047-4a51-928d-33287fa4b20e)

**Comment by PrzemyslawLukanowski** [comment]
@Zavanven Przekazałem problem do zespołu odpowiedzialnego za tę zakładkę. Wrócę z informacją, gdy uda ustalić się więcej szczegółów.

**Comment by PrzemyslawLukanowski** [comment]
@Zavanven Sprawdź proszę teraz, czy możesz wybrać metodę weryfikacji.

**Comment by Zavanven** [comment]
@PrzemyslawLukanowski Potwierdzam, że mogę wybrać metodę weryfikacji. Natomiast dalej nie mogę aktywować konta. W testowym systemie płatności wybieram "zapłać" i pomimo otrzymania maila od przelewy24, płatność dalej nie przechodzi.
![image](https://github.com/user-attachments/assets/ed01d8e3-9189-42bc-9bcf-9a88bbb4ad08)
Otrzymuję takowego maila ale link prowadzi do produkcyjnej wersji allegro. Weryfikacja konta bankowego nie przechodzi po stronie sandboxa.

**Comment by Zavanven** [comment]
@PrzemyslawLukanowski Udało mi się aktywować konto, bądź ktoś to zrobił za mnie. Przyszła mi informacja zwrotna, że rachunek został przypisany. Serdcznie dziękuję za pomoc.

**Comment by PrzemyslawLukanowski** [comment]
@Zavanven Tak, wykonałem wstępną aktywację ręcznie po naszej stronie, nie jestem jednak przekonany, czy możesz już wystawiać oferty, dlatego weryfikuję jeszcze ten temat. Czy mógłbyś sprawdzić, jak to wygląda po Twojej stronie?

**Comment by Karol8181** [comment]
Witam,
Podpinam się pod temat. Proszę o weryfikację konta **ct_sandbox**

**Comment by MaciejFrackowiak** [comment]
@Karol8181 Konto zweryfikowane.
